packages/main/src/plugin/agent-workspace/agent-workspace-manager.ts

```
@@ -768,5 +777,6 @@ export class AgentWorkspaceManager implements Disposable {
       }
     }
     this.workspaceTerminals.clear();
+    this.disposables.forEach(disposable => disposable.dispose());
```

Biome (2.5.3)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)